src/main/java/com/google/api/generator/engine/writer/JavaFormatter.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.writer;
+
+import com.google.googlejavaformat.java.Formatter;
+import com.google.googlejavaformat.java.FormatterException;
+
+public final class JavaFormatter {
+
+  private static final Formatter FORMATTER = new Formatter();
+
+  private JavaFormatter() {}
+
+  public static String format(String comment) {
```

Comment:
Nit: Will the input always be a comment? Consider renaming this parameter to something more generic.

Reply:
thanks, change to `sourceString` fore more general purpose.

Reply:
done.